clients/cli/src/test/java/org/apache/gravitino/cli/integration/test/TableFormatOutputIT.java

```diff
@@ -59,7 +59,9 @@ public void startUp() {
       commandArg(GravitinoOptions.PROPERTIES),
       "jdbc-url=jdbc:postgresql://postgresql-host/mydb,jdbc-user=user,jdbc-password=password,jdbc-database=db,jdbc-driver=org.postgresql.Driver",
       commandArg(GravitinoOptions.URL),
-      gravitinoUrl
+      gravitinoUrl,
+      commandArg(GravitinoOptions.COMMENT),
+      "catalog，用于测试"
```

Comment:
Don't modify original catalog(`postgres`), and I think we can create a new catalog(`postgres2`) have comments
```
    String[] create_catalog2_args = {
      "catalog",
      "create",
      commandArg(GravitinoOptions.METALAKE),
      "my_metalake",
      commandArg(GravitinoOptions.NAME),
      "postgres2",
      commandArg(GravitinoOptions.PROVIDER),
      "postgres2",
      commandArg(GravitinoOptions.PROPERTIES),
      "jdbc-url=jdbc:postgresql://postgresql-host/mydb,jdbc-user=user,jdbc-password=password,jdbc-database=db,jdbc-driver=org.postgresql.Driver",
      commandArg(GravitinoOptions.URL),
      gravitinoUrl
      gravitinoUrl,
      commandArg(GravitinoOptions.COMMENT),
      "catalog，用于测试"
    };
    Main.main(create_catalog2_args);
```

Reply:
done, plz check it again.